### PR TITLE
Voter noise

### LINKED ISSUE #216: Voter dynamics needs more random restarts

### PR DESCRIPTION
**Pull request by sdmccabe**
Resolves #216.

Should 'automatic' or None be the default?

### CODE REVIEW

File: netrd/dynamics/voter.py [hunk 1]
```diff
@@ -37,10 +37,10 @@ def simulate(self, G, L, noise=None):
         L (int)
             the length of the desired time series.
 
-        noise (float or None)
+        noise (float, str or None)
             if noise is present, with this probability a node's state will
             be randomly redrawn from :math:`\{-1, 1\}` independent of its
-            neighbors' states.
+            neighbors' states. If 'automatic', set noise to :math:`1/N`.
```
Comment (leotrs) [inline netrd/dynamics/voter.py]: Why 1/N?

**Comment by leotrs** [comment]
No comments about what the default should be. Probably the one that generates the least surprising behavior.

**Comment by sdmccabe** [comment]
I think `None` is the least surprising, since that's the "true" voter model. Re: 1/N, two answers:

1. That's what we mentioned in the issue.
2. Heuristically, it seemed to produce nice-looking dynamics in testing; i.e., it did not get stuck in all 1 or -1 regimes, while there was still structure (you could see the cliques when running it on the ring of cliques).

**Comment by leotrs** [comment]
I remember we mentioned it. I meant it would be nice to point out in the docstring why `1/N` is a good value.  If it's just a experimental heuristic then I don't think there's much to point out.

**Pull request merged**